pkg/cmd/experimental/diagnostics/cluster.go

```diff
@@ -57,6 +57,10 @@ func (o DiagnosticsOptions) buildClusterDiagnostics(rawConfig *clientcmdapi.Conf
 			diagnostics = append(diagnostics, &clustdiags.ClusterRoles{ClusterRolesClient: clusterClient, SARClient: clusterClient})
 		case clustdiags.ClusterRoleBindingsName:
 			diagnostics = append(diagnostics, &clustdiags.ClusterRoleBindings{ClusterRoleBindingsClient: clusterClient, SARClient: clusterClient})
+		case clustdiags.ValidateMasterConfigName:
+			diagnostics = append(diagnostics, &clustdiags.ValidateMasterConfig{MasterConfigFile: o.MasterConfigLocation})
+		case clustdiags.ValidateNodeConfigName:
+			diagnostics = append(diagnostics, &clustdiags.ValidateNodeConfig{NodeConfigFile: o.NodeConfigLocation})
 
```

Comment:
Don't think these additions are necessary; current host diagnostics do master and node config validation already.